.gitlab-ci.yml

```diff
@@ -509,6 +509,9 @@ library:ci-unimath:
 library:ci-vst:
   <<: *ci-template-flambda
 
+library:ci-verdi-raft:
```

Comment:
Please keep the entries in alphabetical ordering.

Reply:
Fixed.